pkg/builder/builder.go

```diff
@@ -280,7 +290,8 @@ func (b generalBuilder) build(ctx context.Context, pw progresswriter.Writer) err
 				}
 				_, err := b.Client.Build(ctx, solveOpt, "envd", b.BuildFunc(), pw.Status())
 				if err != nil {
-					err = errors.Wrap(err, "failed to solve LLB")
+					err = errors.Wrap(&BuildkitdErr{err: err}, "Buildkit error")
+					logrus.Errorf("%+v", err)
```

Comment:
Use %w to format errors

Reply:
This is intentional. %+v is a feature of `cockroachdb/errors` to print the full error messages including stacktrace.